owlbot.py

```diff
@@ -21,4 +21,8 @@
     s.move(library)
 
 s.remove_staging_dirs()
-java.common_templates()
+java.common_templates(excludes=[
+    '.kokoro/presubmit/graalvm-native.cfg',
```

Comment:
You may want to exclude the graalvm-native cfg from the exclusion list. The Graalvm team is still working on it and may require changes to the file.

Reply:
@Neenu1995 thank you for the comment. This is a cherrypick from what we have on `main`. Should that change go into `main` as well if we are making that change in the `2.2.x` branch?

Reply:
More context, it comes from this PR: https://github.com/googleapis/java-container/pull/628

Reply:
The way I understand it, if we don't exclude the graalvm-native config, then changes to it are going to get reverted by owl bot. In this case here, we want to make changes to it (i.e. moving from `gcloud-devel` to JDST)